frontend/lib/requests/authorized-user.ts

```
@@ -346,33 +351,228 @@ export async function fetchIsAuthorizedUser(email: string) {
+      firstTime,
+      linkedin,
+      github,
+      photo,
+    } = updates;
+    const roleIds = roles
+      ? await Promise.all(
+          roles.map((role) => getAuthorizedUserRoleIdByTitle(role)),
+        )
+      : [];
+
+    const data: any = {};
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)